### DISCUSSION TITLE
Config option to change default sorting

### DISCUSSION THREAD

**Discussion by cowsay1** [umputun/remark42#1630] [Ideas]
It would be nice to have config option to change DEFAULT_SORT from -active to something (-time) without rebuilding from sources.

In my case, there are long threads where only a few people engage in discussions, repeatedly bumping this old thread that started years ago. I don't want this thread to occupy the first five pages or scrolls of comments, causing new comments to be hidden beneath it.

**Comment by geerlingguy** [comment]
Ditto; on my blog I like to sort by 'Oldest' first, because that gives a good ordered approach to the comment history (even if it might not give the most relevant comments for a very very old post...). But it would be nice to set in the `remark_config` a `default_sort` or something like that.
